### DISCUSSION TITLE
Windows versions of the libraries

### DISCUSSION THREAD

**Discussion by olekrisek** [jasonacox/Build-OpenSSL-cURL#75] [General]
As this was a tremendeous help getting those libraries done. Is it possible to have a windows version of this too ? Win64?

**Comment by jasonacox** [comment]
Hi @olekrisek , thanks for the kind feedback. Are you talking about creating iOS binaries or Win64 binaries? In any case, I'm not a Windows developer, but I can only assume it would be a significant rewrite (e.g. moving from bash to powershell). There may be a path using WSL, but I assume the intent is to build binaries and libraries for native Windows development?

Let me what specific outcome you would like to see. Happy to consider.

**Reply by olekrisek** [reply]
Hi @jasonacox , 
Thank you for your replay on this. I am the developer of a FileMaker Plugin, that now works well on MacOS plattforms. However, some users use Windows clients, and cannot use the plugin. I would like to port the plugin to Windows,  so all libraries has to be recompiled for that platform. Having cURL with OpenSSL shared with the boost libraries also using the same OpenSSL (mySQL, etc) would be good. I am also using ICU libs but that is another case.
